src/routes/transform_photos.py

```diff
@@ -0,0 +1,63 @@
+from typing import Type, Tuple
+
+from cloudinary import CloudinaryImage
+from fastapi import APIRouter, status, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+from src.database.db import get_db
+from src.database.models import Photo, User
+from src.repository.users import get_current_user
+from src.schemas import TransformedPhotoModelResponse, TransformPhotoModel
+from src.enums import Roles
```

Comment:
from src.enums import Roles
from src.security.role_permissions import RoleChecker

Are not in use.